ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1146,8 +1146,11 @@ def update_logging(self):
         )
         entry = ldap.get_entry(dn)
         entry.single_value["nsslapd-logging-hr-timestamps-enabled"] = 'off'
-        ldap.update_entry(entry)
-
+        try:
+            ldap.update_entry(entry)
+        except errors.DatabaseError:
```

Comment:
Not a deal breaker, just a question. Does the DatabaseError include a string which would identify that the option is unavailable? Should we test for that too in order to avoid masking other possible DatabaseError exceptions?

Reply:
My new patch ensures that the DatabaseError is related to "Unknown attribute nsslapd-logging-hr-timestamps-enabled".
I also added a restart of dirsrv server in the test for cert expiration. Otherwise the test is often failing with a 389ds coredump and I suspect it happens because DS doesn't like large time jumps.